<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTorrent</class>
 <widget class="QDialog" name="AddTorrent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add torrent</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Torrent</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Torrent:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLineEdit" name="TorrentFile_"/>
          </item>
          <item>
           <widget class="QPushButton" name="TorrentBrowse_">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Files:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTreeView" name="FilesView_">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="rootIsDecorated">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QPushButton" name="MarkAll_">
              <property name="text">
               <string>Mark all</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="UnmarkAll_">
              <property name="text">
               <string>Unmark all</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="MarkSelected_">
              <property name="text">
               <string>Mark selected</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="UnmarkSelected_">
              <property name="text">
               <string>Unmark selected</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Destination:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="Destination_"/>
            </item>
            <item>
             <widget class="QPushButton" name="DestinationBrowse_">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="AvailSpaceLabel_">
              <property name="text">
               <string>unknown</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="AvailSpaceBar_">
              <property name="value">
               <number>0</number>
              </property>
              <property name="format">
               <string>%p% used</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Tags:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="LeechCraft::Util::TagsLineEdit" name="TagsEdit_">
          <property name="text">
           <string>untagged</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Add as:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="AddTypeBox_">
          <item>
           <property name="text">
            <string>Started</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Paused</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="TryLive_">
        <property name="text">
         <string>Try to play this torrent live</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Info</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Tracker URL:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="TrackerURL_">
        <property name="text">
         <string>&lt;unknown&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="Size_">
        <property name="text">
         <string>&lt;unknown&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Creator:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="Creator_">
        <property name="text">
         <string>&lt;unknown&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Comment:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="Comment_">
        <property name="text">
         <string>&lt;unknown&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="Date_">
        <property name="text">
         <string>&lt;unknown&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LeechCraft::Util::TagsLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">util/tags/tagslineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="bittorrentresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>OK_</sender>
   <signal>released()</signal>
   <receiver>AddTorrent</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cancel_</sender>
   <signal>released()</signal>
   <receiver>AddTorrent</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TorrentFile_</sender>
   <signal>returnPressed()</signal>
   <receiver>OK_</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Destination_</sender>
   <signal>returnPressed()</signal>
   <receiver>OK_</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TagsEdit_</sender>
   <signal>returnPressed()</signal>
   <receiver>AddTorrent</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
